controller/users.go

```diff
@@ -23,7 +23,7 @@ import (
 	goajwt "github.com/goadesign/goa/middleware/security/jwt"
 	"github.com/jinzhu/gorm"
 	errs "github.com/pkg/errors"
-	"github.com/satori/go.uuid"
+	uuid "github.com/satori/go.uuid"
```

Comment:
Not sure why you need this ( uuid "github.com/satori/go.uuid" == "github.com/satori/go.uuid") but OK :)

Reply:
yeah, let me reverse it, actually.

Reply:
Something wrong with your VSCode then ;) I don't have this problem in mine. But that's a minor thing. We can keep it.